services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -212,9 +216,62 @@ def engine(self) -> Engine:
             async for row in conn.execute(query):
                 result_dict = dict(row.items())
                 log.debug("found project: %s", result_dict)
+                tags = await self._get_tags_by_study(study_id=result_dict['id'])
+                result_dict['tags'] = tags
                 projects_list.append(_convert_to_schema_names(result_dict))
         return projects_list
 
+    async def _get_study(self, user_id: str, project_uuid: str, exclude_foreign: List = []) -> Dict: # pylint: disable=dangerous-default-value
+        async with self.engine.acquire() as conn:
+            joint_table = user_to_projects.join(projects)
+            query = select([projects]).select_from(joint_table).where(
+                and_(projects.c.uuid == project_uuid,
+                     user_to_projects.c.user_id == user_id)
+            )
+            result = await conn.execute(query)
+            study_row = await result.first()
```

Comment:
is there a reason why we use first instead of one?

Reply:
I think there is not a ``one()`` method for ``ResultProxy``. There is the ``fetchOne`` but that creates kind of a cursor to go over several rows. The ``uuid`` column is set as unique so there will never be more than one row.